lib/feed/bloc/feed_bloc.dart

```diff
@@ -231,15 +231,14 @@ class FeedBloc extends Bloc<FeedEvent, FeedState> {
   Future<void> _onFeedItemUpdated(FeedItemUpdatedEvent event, Emitter<FeedState> emit) async {
     emit(state.copyWith(status: FeedStatus.fetching));
 
-    List<PostViewMedia> updatedPostViewMedias = state.postViewMedias.map((PostViewMedia postViewMedia) {
+    for (final (index, postViewMedia) in state.postViewMedias.indexed) {
       if (postViewMedia.postView.post.id == event.postViewMedia.postView.post.id) {
-        return event.postViewMedia;
-      } else {
-        return postViewMedia;
+        state.postViewMedias[index] = event.postViewMedia;
       }
-    }).toList();
+    }
 
-    emit(state.copyWith(status: FeedStatus.success, postViewMedias: updatedPostViewMedias));
+    //emit(state.copyWith(status: FeedStatus.success, postViewMedias: updatedPostViewMedias));
```

Comment:
We can remove this comment if its not being used!

Reply:
Oops, sorry I missed that! All set!